docs/conf.py

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env python3
-# -*- coding: utf-8 -*-
```

Comment:
Goodness, what a time to be alive to see those go finally. :sweat_smile:

Reply:
![you](https://user-images.githubusercontent.com/181628/71658595-f7677500-2d44-11ea-9f34-84d9a0beee1c.png)